template/cordova/lib/run.js

```diff
@@ -155,7 +155,8 @@ module.exports.help = function () {
     console.log('    --appx=<8.1-win|8.1-phone|uap>');
     console.log('                  : Overrides windows-target-version to build Windows 8.1, ');
     console.log('                              Windows Phone 8.1, or Windows 10.');
-    console.log('');
+    console.log('    --win10tools  : Uses Windows 10 deployment tools (used for a Windows 8.1 app when');
```

Comment:
Because Windows 8.1 tools currently can't deploy to an 8.1 app to a Windows 10 device.  Hence the parenthetical and the following line.